### ISSUE TITLE
[Feature] Extract 4844 module to standalone crate in alloy/eips

### ISSUE THREAD

**Issue by mattsse** [alloy-rs/alloy#1821]
### Component

consensus, eips, genesis

### Describe the feature you would like

this alloy-eips crate is a bit bloated and we'd like to provide all 4844 types without relying on this repository like we did with accesslists and 7702:

https://github.com/alloy-rs/eips/tree/main/crates

## TODO
* introduce a new crate in https://github.com/alloy-rs/eips/ `4844`
* duplicate all 4844 code there https://github.com/alloy-rs/alloy/blob/26e52dc82ba85f654a38628b0ce406c75cf6d37c/crates/eips/src/eip4844/mod.rs#L1-L1
* once published remove 4844 from alloy but keep re-export

**Comment by nadtech-hub** [comment]
✋

**Comment by stevencartavia** [comment]
@nadtech-hub hello, are you still working on this?

**Comment by nadtech-hub** [comment]
@stevencartavia yes, I am. Is it taking too long on your sight or you want to take on it?

**Comment by stevencartavia** [comment]
it's fine, let me know in a couple of days to respect your time because idk how long it would take me too! :)

**Comment by nadtech-hub** [comment]
It'll probably take longer for me to complete it

**Comment by Abhishekkochar** [comment]
@nadtech-hub, I would like to take this one if not WIP.

**Comment by nadtech-hub** [comment]
It is in review

**Comment by yash-atreya** [comment]
Blocked by https://github.com/alloy-rs/eips/pull/27.

**Comment by jenpaff** [comment]
> Blocked by [alloy-rs/eips#27](https://github.com/alloy-rs/eips/pull/27).

still need to figure out what exactly we need here

**Comment by nadtech-hub** [comment]
@yash-atreya why did you mark it as blocked?

**Comment by nadtech-hub** [comment]
@jenpaff I wonder what you have figured out.
